datasets/amazon-last-mile-challenges.yaml

```diff
@@ -1,5 +1,5 @@
-Name: 2021 Amazon Last Mile Routing Research Challenge Dataset
-Description: The 2021 Amazon Last Mile Routing Research Challenge was an innovative research initiative led by Amazon.com and supported by the Massachusetts Institute of Technology’s Center for Transportation and Logistics. Over a period of 4 months, participants were challenged to develop innovative machine learning-based methods to enhance classic optimization-based approaches to solve the travelling salesperson problem, by learning from historical routes executed by Amazon delivery drivers. The primary goal of the Amazon Last Mile Routing Research Challenge was to foster innovative applied research in route planning, building on recent advances in predictive modeling, and using a real-world problem and data.
+Name: 2021 Amazon Last Mile Routing Research Challenge Data Set
```

Comment:
Sorry to be pedantic, but we use "dataset" in these parts. :) Please revert this change.